backend-shared/divesummary.cpp

```diff
@@ -111,7 +118,9 @@ void diveSummary::calculateDive(int inx, struct dive *dive)
 		divesDeep[inx]++;
 
 	// sum SAC, check for new min/max.
-	if (dive->sac) {
+	// Do not include dives with SAC > 40 l/min (probably a free flow regulator)
+	// and SAC < 4 l/min (probably a faulty transmittor)
+	if (dive->sac <= 40000 && dive->sac >= 4000) {
```

Comment:
I have several dives with a SAC below 7l, a very good friend of mine has many dives around 5l (she is not a big person). And as a tech dive master, teaching advanced classes in ~8C water, I have seen many students with SAC rates north of 35l
So again. No. We should not have such arbitrary cut-offs. If a user wants to exclude a dive from the summary, give them a way to do so. Do not arbitrary exclude them.
NAK